app/src/main/java/com/example/playlistmaker/ActivitySearch.kt

```diff
@@ -95,6 +103,46 @@ class ActivitySearch : AppCompatActivity() {
         outState.putString(STRING_EDIT_TEXT, stringEditText)
     }
 
+    private fun searchTracks() {
+        if (stringEditText.isNotEmpty()) {
```

Comment:
```suggestion
        if (stringEditText.isEmpty()) return
```
Если написать так, то читать проще